README.md

```diff
@@ -201,6 +206,15 @@ This plugin creates device plugin endpoints based on the configurations given in
                     "vendors": ["8086"],
                     "devices": ["0d90"]
             }
+        },
+        {
+            "resourceName": "bf2_sf",
```

Comment:
things like this break in next 4 months with bf3.

There is nothing special about SF of bf2 vs bf3.

Please remove bf2_ prefix.

Reply:
@DmytroLinkin better to have more practical names that users can copy.
Otherwise they will copy bf2_sf and it will fail when they upgrade sw to new generation asic.

Reply:
It would not fail because it is not HW/SW dependent. Resource name is given by the admin.